src/AddedTasks.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import './AddedTasks.css';
+import deletePng from './images/delete.png';
+import editPng from './images/edit.png';
+
+export default class AddedTasks extends Component {
+  render() {
+    const tasks = this.props.tasks;
+    
+    let tasksElements = tasks.map(task => {
+      return (
+        <div className="added-task_container" key={task.id} id={task.id}>
+        <span className="added-task_span">{task.name} | {task.description}</span>
+        <div className="image-container">
+          <img className="img-edit" title="Edit" src={editPng} alt="edit" onClick={this.editAddedTask}/>
+          <img className="img-delete" title="Remove" src={deletePng} alt="delete" onClick={this.getIdRemovedTask}/>
+        </div>
+        </div>
+      );
+    });
+    
+    return (
+      <div>
+        {tasksElements}
+      </div>
+    );
+  };
+
+  editAddedTask = e => {
+    e.preventDefault();
+    
+    const showEditTaskForm = this.props.showEditTaskForm;
+    const getDataEditableTask = this.props.getDataEditableTask;
+    const editTaskId = e.target.parentElement.parentElement.id;
```

Comment:
better to take id from props.tasks[<your task>].id. If you create independent component for <Task> then get id from the task will be much easier.